### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by marceloaof**
Hello❗(emoji extracted from Awesome Emoji Picker)

I am creating the translation for Brazilian Portuguese as a pull request, according to what was required on https://github.com/rugk/awesome-emoji-picker/issues/107

This is my first pull request on GitHub, so please inform me if any errors were made.

@rugk, I translated the add-on listings on AMO as well, after you mentioned it.

### CODE REVIEW

**Comment by rugk** [comment]
Hi @marceloaof,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Thanks for your translation.
Please subscribe to #27 to get notified for new updates on translations etc. :smiley:

I'll review this soon.

File: src/_locales/pt-BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
```
Comment (rugk) [inline src/_locales/pt-BR/messages.json:402]: I guess you should add yourself here (and link to you above). See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on

**Review by rugk — CHANGES_REQUESTED** [review]
Only one small change. Otherwise LGTM.

So congratulations to your first PR! :smiley:

**Comment by marceloaof** [comment]
Vielen Dank for your guidance, @rugk 😘 Now I understand my errors on GitHub thanks to you.

Following your suggestion, I have subscribed myself to receive notifications about new translations. Therefore, I will be able to assist in translating the project when it is available to translate the names of the emojis. 📝 Recently, [other project](https://github.com/vector-im/element-web/issues/11013) was able to achieve that. I inform you in the hope to advance that scenario here.

Rugk, what on German do you mean by hacking on it? 🙃 Hab viel Spass?

**Comment by rugk** [comment]
Well hacking on it is just what you do, contributing to it. Was just little colloquial term [referring to the original meaning](https://de.wikipedia.org/wiki/Hacker).

**Review by rugk — APPROVED** [review]

**Comment by rugk** [comment]
Ah BTW another question: Could we use it for `pt` (i.e. portguese, all variants) or does it need to be only shown for `pt-BR`?

**Review by rugk — APPROVED** [review]
Requesting a potential review from @CGReinhold (Portuguese translator in Offline QR Code). Unless there are objections, this will be merged soon.

**Review by CGReinhold — APPROVED** [review]

**Comment by marceloaof** [comment]
Hello, @rugk
👍 Undoubtedly, reading Brazilian Portuguese on this add-on is a better experience for natives of Portugal, Angola and other countries than English currently is.
👉 However, while people say in Portuguese from Portugal (and in German) that "du kannst und das Add-on kann", in Brazilian Portuguese people say "du kann und und Add-on kann".
🔜 The translation I sent is understandable to Portuguese readers, but some nuances will be noticeable to non-Brazilians.
You may either offer the translation as portuguese in general (pt), making the distinction in the future, or offer it duplicated (pt-br + pt). A native from Portugal will need to make low priority corrections for the translation granted to Portugal.

**Comment by rugk** [comment]
Well, what is/are your suggestions, @marceloaof and @CGReinhold? As I don't know this language I don't know how big the distraction would be to offer it as `pt`.

**Comment by CGReinhold** [comment]
I don't think that it would be a big problem offering it as `pt` since the differences are not big and most softwares already have only the brazilian portuguese option. Since there isn't currently a Portuguese translator, I believe that this is the way to go.

**Review by CGReinhold — APPROVED** [review]

**Pull request merged**